SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,90 @@
+
+# `spdlog` Quality Declaration
+
+TODO: Declare the quality level for `spdlog`
+
+## Version Policy
+
+### Version Scheme
+
+There is a three-part version attached to `spdlog` releases.
+The sources define the version parts as `MAJOR`.`MINOR`.`PATCH`.
+There is an old ticket where the project's primary developer discusses versioning, but there is no documentation officially stating what approach the developer is taking.
+
+### API/ABI Stability Within a Released ROS Distribution
+
+As an external package, `spdlog` is not released on the same cadance as ROS.
+However, the `spdlog_vendor` package controls what version of `spdlog` is acceptable, so API/ABI stability within a ROS release can be enforced.
+
+### Public API Declaration
+
+Because `spdlog` can be consumed as a header-only library, nearly all of the source code can be considered part of the public API.
+For the compiled portion of `spdlog`, the `SPDLOG_API` C++ macro indicates the symbols that are part of the public API.
+
+## Change Control Process
+
+The source code repository for `spdlog` is hosted in Github, and the change process follows a typical Github pull request model.
+It appears that a single developer is reviewing and merging the proposed changes.
+The developer appears to be active in triaging incoming issues and addressing incoming pull requests in a timely manner.
+
+Incoming pull requests automatically trigger continuous integration testing across Linux and macOS, and must run successfully to be merged.
+
+## Documentation
+
+### Feature Documentation
+
+A feature list is included in the `spdlog` documentation in the repository.
+
+### Public API Documentation
+
+The API for `spdlog` is documented with a targeted example for each advertised feature.
+This documentation is available in the repository, and there is more detailed information on some specific topics available in the Github Wiki for `spdlog`.
+
+### License
+
+The license for `spdlog` is MIT, and all of the code includes a header stating so.
+The `spdlog` repository also includes a `LICENSE` file with the full terms.
+
+There is some `spdlog` test code which is licensed under Boost 1.0, and the full license text is included alongside the affected source code files.
+
+### Copyright Statements
+
+Each of the `spdlog` source files containing code include a copyright statement with the license information in the file's header.
+
+## Testing
+
+### Feature Testing
+
+Many (if not all) of the advertised features of `spdlog` are tested as part of continuous integration.
+
+### Public API Testing
+
+There is some, but very little testing in `spdlog` specifically focused on API stability.
+
+### Coverage
+
+There is no test coverage tracking in `spdlog`.
+
+### Performance
+
+Performance tests for `spdlog` can be built by specifying the `SPDLOG_BUILD_BENCH` CMake option.
```

Comment:
Is there a link to add for this so someone can jump straight to its docs or current status?